### ISSUE TITLE
[Bug] Turbomind random crash when handle batch request

### ISSUE THREAD

**Issue by tuilakhanh** [InternLM/lmdeploy#4453]
### Checklist

- [x] 1. I have searched related issues but cannot get the expected help.
- [x] 2. The bug has not been fixed in the latest version.
- [x] 3. Please note that if the bug-related issue you submitted lacks corresponding environment info and a minimal reproducible demo, it will be challenging for us to reproduce and resolve the issue, reducing the likelihood of receiving feedback.

### Describe the bug

Crash with log:
`[TM][FATAL] kernels/apply_token_bitmask_inplace_cuda.cu(212): Check failed: logits_shape.first ==bitmask_shape.first logits and bitmask must have the same batch size.`

### Reproduction

Sending a batch of approximately 10 requests or higher.
Randomly, TM will crash.

### Environment

```Shell
*
```

### Error traceback

```Shell

```

**Comment by windreamer** [comment]
@tuilakhanh Could you kindly try #4456 to check if it fixes the issue?

**Comment by tuilakhanh** [comment]
Fixed with https://github.com/InternLM/lmdeploy/pull/4456
